instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/netty/v4_1/server/HttpServerResponseTracingHandler.java

```diff
@@ -103,6 +103,10 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise prm) {
       span.setStatus(code >= 100 && code < 500 ? StatusCode.UNSET : StatusCode.ERROR);
     }
     if (msg instanceof LastHttpContent) {
+      ctx.channel()
```

Comment:
Sounds like this could use some comment?

Reply:
Added